### ISSUE TITLE
makeRequest is not using the headers in swift3 client

### ISSUE THREAD

**Issue by ahmettaha** [swagger-api/swagger-codegen#5251]
The code below ignores the headers in AlamofireImplementations. 

```
open func makeRequest(manager: SessionManager, method: HTTPMethod, encoding: ParameterEncoding) -> DataRequest {
        return manager.request(URLString, method: method, parameters: parameters, encoding: encoding)
    }
```

I believe the code should be like below. Am I missing something?

```
 open func makeRequest(manager: SessionManager, method: HTTPMethod, encoding: ParameterEncoding) -> DataRequest {
        return manager.request(URLString, method: method, parameters: parameters, encoding: encoding, headers: headers)
    }

```

**Comment by wing328** [comment]
@ahmettaha thanks for reporting the issue. I wonder if you can file a PR so that we can review the fix.

A good starting point is https://github.com/swagger-api/swagger-codegen/blob/master/modules/swagger-codegen/src/main/resources/swift3/AlamofireImplementations.mustache#L48

**Comment by wing328** [comment]
Closed via #5314
